docker-compose.yml

```diff
@@ -26,7 +27,7 @@ services:
     volumes:
       - celestia-app:/home/celestia/.celestia-app
     command: >
-      start --force-no-bbr
+      start --force-no-bbr --api.enable true --api.address tcp://0.0.0.0:1317
```

Comment:
Could we adapt this to rather than providing start flags, override the config directly in the `init.sh` used by the initialisation container?

See here: https://github.com/celestiaorg/celestia-zkevm-hl-testnet/blob/main/testnet/celestia-app/init.sh#L15-L17

Similarly to how we enable grpc, we can simply extend it like:

```bash
celestia-appd config set app api.enable true
celestia-appd config set app api.address tcp://0.0.0.0:1317
```